packages/portal/src/mixins/advancedSearch.js

```diff
@@ -95,17 +101,23 @@ export default {
   },
 
   methods: {
+    advancedSearchFieldByName(name) {
+      return this.advancedSearchFields.find((field) => field.name === name);
+    },
```

Comment:
Add a blank line after each of the new methods for consistency with what pre-existed.